test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -727,6 +782,118 @@ public void CreateDisplayMetadata_IsFlagsEnum_ReflectsModelType(Type type, bool
                 .ThenBy(item => item.Key.Name, StringComparer.Ordinal));
         }
 
+        private DataAnnotationsMetadataProvider CreateLocalizingProvider()
```

Comment:
Please move this down to the bottom of the class, perhaps just above the nested classes. And rename it to mention `IStringLocalizer`. Current name is too broad.